### PR TITLE
[FLINK-25278][ci] Use Maven proxy for confluent repo

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: tools/ci/alibaba-mirror-settings.xml [hunk 1]
```diff
@@ -24,5 +24,11 @@ under the License.
       <url>http://172.17.0.1:8888/repository/maven-central/</url>
       <mirrorOf>central</mirrorOf>
     </mirror>
+    <mirror>
+      <id>alicloud-mvn-confluent-mirror</id>
+      <name>Alibaba Confluent Maven mirror</name>
+      <url>http://172.17.0.1:8888/repository/confluent/</url>
```
Comment (dmvk) [inline tools/ci/alibaba-mirror-settings.xml:30]: Does Alibaba provide the https variant of the maven proxy?

Also do they have a DNS record available for the proxy by any chance?

**Review by dmvk — APPROVED** [review]
LGTM, thanks for the PR Chesnay 👍 I've added two minor questions, PTAL

File: tools/ci/alibaba-mirror-settings.xml (hunk 1 above)
Comment (zentol) [reply tools/ci/alibaba-mirror-settings.xml:30]: This is something that we have setup ourselves on the machines, so no to both.

Comment (dmvk) [reply tools/ci/alibaba-mirror-settings.xml:30]: 👍

**Pull request merged**